packages/network-of-terms-catalog/catalog/datasets/filmmuseum-genres.rq

```diff
@@ -0,0 +1,37 @@
+{

```

Comment:
This file should have extension jsonld, not rq.

Reply:
sorry. that was a bit sloppy of me. should be fixed.